### ISSUE TITLE
Review Helmet config

### ISSUE THREAD

**Issue by hrmoller** [DBCDK/serviceprovider#563]
The use of Helmet should be reviewet and maybe optimised.

**Comment by hyatt03** [comment]
Helmet is a library for setting http headers that help the browser in protecting the client, it's composed of 10 functions that can be used individually.

* helmet.contentSecurityPolicy: Very relevant for the consuming site to implement, but it doesn't help us.
* xssFilter: The xss filter protects against a subset of the problems mitigated by CSP. We can't really use it as it needs to be implemented by the consuming site.
* frameguard: The frameguard prevents or limits the use of iframes and frames, this can reduce the risk of clickjacking. This isn't very relevant for us as an api.
* hsts: Prevents the use of http, this is a very good idea, also for us! But this is already handled by the outer layers (specifically our proxy), so it's just a resource waste.
* hidePoweredBy: This function disables the X-Powered-By header, this is already handled by using `app.disable('x-powered-by');`
* ieNoOpen: This feature prevents untrusted html to be opened by IE in the context of our site. This is another feature mostly relevant for the consumer.
* noSniff: This feature prevents mime type sniffing, and therefore protects against text files being executed as JS and the like. This is another feature mostly relevant for the consumer.
* noCache: Prevents the caching of html pages and static resources, this is pretty relevant for us.
* hpkp: HTTP Public Key Pinning ensures that your certificates cannot be forged by requiring the client to cache the public key. This prevents Man in the middle attacks. This is also very relevant for us, but like hsts, it's already handled by outer layers.
* dnsPrefetchControl: Disables dns prefetching to improve security, yet another feature already implemented by our proxy.

In summary, the only feature really relevant for us to implement in the serviceprovider is noCache, but this is easily set with express alone, and therefore I'll be removing helmet.
